frontend/src/pages/modelRegistrySettings/utils.ts

```diff
@@ -51,3 +51,20 @@ export const constructRequestBody = (
 
   return mr;
 };
+
+export const isClusterWideCABundleEnabled = (
```

Comment:
that this may require unit tests but the coverage didn't catch this.. so we can maybe ignore.. the cypress tests cover this scenario.